CHANGELOG.md

```diff
@@ -21,7 +21,12 @@ Enables the use of pre-release builds of Omnisharp. Downloading a pre-release bu
 
 #### Editor
 
-* Updated the Omnisharp Status Bar to appear on left instead of right because it displays the project level info(which is a characteristic of the elements on the left side of the window) and removed ambiguities in its behavior.([#2146](https://github.com/OmniSharp/omnisharp-vscode/issues/2146), PR: [@2133](https://github.com/OmniSharp/omnisharp-vscode/pull/2133))
+* Splits the OmniSharp status bar in two parts both of which appear on the left and have specific responsibilities. ([#2146](https://github.com/OmniSharp/omnisharp-vscode/issues/2146), PR: [@2133](https://github.com/OmniSharp/omnisharp-vscode/pull/2133))
+  * Omnisharp Server Status Bar - Shows the various states that the OmniSharp server initialisation might be in(eg: Downloading, Installing, etc). The flame would be green when the server is initilaised and running properly and red if there is an error.
+  * Project Information Status Bar - 
+    * It will display the workspace level information irrespective of the document in the active text editor.  
+    * If a project is already selected, the status bar would display the name of the currently selected project and clicking on which would display a menu to switch other projects in the workspace. 
+    * If there are multiple possible launch targets, it would display 'Select Project', clicking on which would display a menu to select one.
```

Comment:
* it would display -> it **displays** _(incorrect tense).
* 'Select Project', clicking -> 'Select Project'**. Clicking** _(replace comma with period and begin new sentence)_
* on which would display -> on **it displays** _(word choice; incorrect tense)_